lib/index.js

```diff
@@ -5,14 +5,15 @@ const Checker = require("./check")
 
 const timezone = core.getInput('timezone')
 const calendar = core.getInput('calendar')
-const schedule = YAML.parse(core.getInput('schedule'))
+const schedule = YAML.parse(core.getInput('schedule', {required: true}))
 const check_future_days = parseInt(core.getInput('check_future_days'))
 const fail_on_failure = core.getBooleanInput('fail_on_failure')
+const types = core.getInput('types').split(',').filter(x => x !== "")
```

Comment:
```suggestion
const types = core.getInput('types').split(',').filter(holidayType => holidayType  !== "")
```
question: why do you need to filter function? It just handle when comma is at the end of string `public,bank,optional,school,`

Reply:
Deleted